content/backmatter.tex

```diff
@@ -405,36 +395,36 @@ \subsection{Header Directory: \HEADER{mpp}}
 include 'mpp/shmem.fh'
 include 'mpp/shmemx.fh'
 \end{lstlisting}
-for backwards compatibility with SGI SHMEM.
+for backwards compatibility with \ac{SGI} SHMEM.
 
-\subsection{\CorCpp: start\_pes}
+\subsection{\CorCpp: \FUNC{start\_pes}}
 The \CorCpp routine \FUNC{start\_pes} includes an unnecessary initialization
 argument that is remnant of historical \emph{SHMEM} implementations and no
 longer reflects the requirements of modern \openshmem implementations.
 Furthermore, the naming of \FUNC{start\_pes} does not include the standardized
 \shmemprefixLC{} naming prefix. This routine has been deprecated and
-\openshmem users are encouraged to use
-\hyperref[subsec:shmem_init]{\FUNC{shmem\_init}} instead.
-
-\subsection{\CorCpp: \_my\_pe, \_num\_pes, shmalloc, shfree, shrealloc, shmemalign}
-The \CorCpp routines \FUNC{\_my\_pe}, \FUNC{\_num\_pes}, \FUNC{shmalloc},
-\FUNC{shfree}, \FUNC{shrealloc} and \FUNC{shmemalign} were deprecated in order
-to normalize the \openshmem \ac{API} to use \shmemprefixLC{} as the standard
-prefix for all routines.
+\openshmem users are encouraged to use \FUNC{shmem\_init} instead.
 
 \subsection{Implicit Finalization}
 Implicit finalization has been replaced with explicit finalization using the
 \FUNC{shmem\_finalize} routine.  Explicit finalization improves portability and
 also improves interoperability with profiling and debugging tools.
 
-\subsection{Fortran: START\_PES, MY\_PE, NUM\_PES}
+\subsection{\CorCpp: \FUNC{\_my\_pe}, \FUNC{\_num\_pes}, \FUNC{shmalloc},
+    \FUNC{shfree}, \FUNC{shrealloc}, \FUNC{shmemalign}}
+The \CorCpp routines \FUNC{\_my\_pe}, \FUNC{\_num\_pes}, \FUNC{shmalloc},
+\FUNC{shfree}, \FUNC{shrealloc}, and \FUNC{shmemalign} were deprecated in order
+to normalize the \openshmem \ac{API} to use \shmemprefixLC{} as the standard
+prefix for all routines.
+
+\subsection{Fortran: \FUNC{START\_PES}, \FUNC{MY\_PE}, \FUNC{NUM\_PES}}
 The \Fortran routines \FUNC{START\_PES}, \FUNC{MY\_PE}, and \FUNC{NUM\_PES}
 were deprecated in order to minimize the API differences from the deprecation
 of \CorCpp routines \FUNC{start\_pes}, \FUNC{\_my\_pe}, and \FUNC{\_num\_pes}.
 
-\subsection{Fortran: SHMEM\_PUT}
-The \Fortran function \FUNC{SHMEM\_PUT} is defined only for the \Fortran
-\ac{API} and is semantically identical to \Fortran functions
+\subsection{Fortran: \FUNC{SHMEM\_PUT}}
```

Comment:
`Fortran` -> `\Fortran`? Although, this may break the PDF TOC entry here, which means it should be `\textit{Fortran}`.